addons/l10n_ph/models/res_partner.py

```diff
@@ -19,4 +20,20 @@ def _commercial_fields(self):
         return super()._commercial_fields() + ['branch_code']
 
     def check_vat_ph(self, vat):
-        return len(vat) == 17 and CHECK_VAT_PH_RE.match(vat)
+        return len(vat) == 11 and CHECK_VAT_PH_RE.match(vat)
```

Comment:
It would be good to check why we had these 17 before, because this means that every time someone will want to change something in existing dbs it will fail.  (Maybe we could support both)

Reply:
The different fields on res_partner are missing the l10n_ph prefix and we should have questioned them more.  What is the reason for first name, middle name and last name?